compiler/damlc/daml-visual/src/DA/d3-dev/README.md

```diff
@@ -0,0 +1,29 @@
+### Dev Setup
+The dev setup is not automated, to get a local dev env follow steps.
+
+    cd compiler/damlc/daml-visual/src/DA/d3-dev
+    wget https://github.com/alexandersimoes/d3plus/releases/download/v1.9.8/d3plus.zip
+    unzip d3plus.zip
+    python -m http.server --directory .
+    open http://0.0.0.0:8000/d3-dev.html
+
+If there are any changes to be made to the JS, they will have to be made to the webpage template within `Visual.hs` module.
+
+Note: We can automate this with using npm/yarn but is not done as of yet as we are not dealing with a lot of JS/HTML/CSS code. If we add more customization it might be worth the effort to setup a automated dev setup.
+
+### About D3 and D3Plus and versions used in this project
+[D3JS](https://d3js.org/) is a popular Data visualization library and [D3Plus](https://github.com/alexandersimoes/d3plus) gives simple api with reasonable defaults to draw SVG.
+
+The version currently we are using is `1.9.8` and the next version of `D3Plus` is `2.0.0` which takes compositional style i.e. lets us pick the libs (tooltips, legends etc.) that we need. The reason we are not using latest(`2.0.0`) is the composition of libs is not very well documented and lacks advanced examples and also it is currently labeled as beta.
```

Comment:
I’m not entirely convinced this is a good idea. Starting out by already lagging behind the latest version seems like a good way of creating tech debt.

Reply:
I agree. I spent two days trying to use the latest version but matching up version of libs and the base libs are not documented yet there are some pointers on google mailing list but I failed to reproduce it  (https://github.com/d3plus/) the ones we are interested are tooltips, legends, and network libs. Authors did not mention release dates so I ended up using the current stable version. Once it is released it should be a hard thing to integrate